demo.html

```diff
@@ -0,0 +1,262 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>Demo Grapher</title>
+    </head>
+    <body>
+        <p>
+            Enter a JS math equation<br>using x, y, k, and theta:
+        </p>
+        <div style="color: white; z-index: 5;">
+            <input style="border-radius: 1rem;" id="ieq" value="Math.sin(k*x)*Math.cos(y/k)" type="text">
+            <br>
+            <button style="border-radius: 1rem;" onclick="handleInput();" type="button" id="1">Save</button>
+            <button style="border-radius: 1rem; display:none;" onclick="cancel(); handleInput();" type="button" id="2">Save</button>
+        </div>
+        <canvas style="width: 80%; position: absolute; top: 50%; left: 50%; transform: translate(-50%, -50%);" id="bg">Start streaming on a modern browser!</canvas>
+    </body>
+    <script>
```

Comment:
Since there's so much JS code, it might be easier to just put this in a separate JS file, like `demo.js` and move the files to `demo/`?
```suggestion    
    <script src="demo.js" type="text/javascript">
```

Reply:
@Quantalabs can you do this?

Reply:
I'll do it

Reply:
I'll do it

Reply:
Okay, ping me so that we can merge when this issue is resolved

Reply:
Done